src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -1241,16 +1246,14 @@ void handle_object_removed(LocalSchedulerState *state,
 }
 
 int num_waiting_tasks(SchedulingAlgorithmState *algorithm_state) {
-  task_queue_entry *elt;
-  int count;
-  DL_COUNT(algorithm_state->waiting_task_queue, elt, count);
+  TaskQueueEntry *elt;
```

Comment:
Hmm I think we still need it for testing purposes (definition of `SchedulingAlgorithmState` isn't exposed).